### PR TITLE
Shortlinks Calls

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Source file for link.tomesh.net

### CODE REVIEW

File: communications/shortlinks.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+cloud,https://cloud.tomesh.net
+github,https://github.com/tomeshnet
+gh,https://link.tomesh.net/tomeshnet
+chat,https://chat.tomesh.net/#/room/#tomesh:tomesh.net
+tasks,https://github.com/tomeshnet/toronto-community-network/projects/1
+new-task,https://github.com/tomeshnet/toronto-community-network/issues/new?template=task_template.md
+teams,https://github.com/orgs/tomeshnet/teams
+passwords,https://pass.tomeshnet.coop
+calls,https://bbb.hypha.coop/b/tom-v6m-hyj
+general-calls,https://bbb.hypha.coop/b/tom-ygc-ttm
```
Comment (makeworld-the-better-one) [inline communications/shortlinks.csv:12]:
```suggestion
meeting,https://bbb.hypha.coop/b/tom-v6m-hyj
call,https://bbb.hypha.coop/b/tom-ygc-ttm
```

Comment (darkdrgn2k) [reply communications/shortlinks.csv:12]: Can i counter like this

- calls - they are not necessary meetings, its a general purpose room. But additional bunch of us will have to remember multiple different short codes.  
- meetings - for official tomesh general meetings?

```suggestion
calls,https://bbb.hypha.coop/b/tom-v6m-hyj
meetings,https://bbb.hypha.coop/b/tom-ygc-ttm
```

Comment (benhylau) [reply communications/shortlinks.csv:12]: We can also name rooms. Like private-1 public-1, or physical places like subway stop names like blood dundas. calls and meetings is more confusing as they mean the same thing. In offices we'd give names to meeting rooms like, we can do the same here.

Comment (makeworld-the-better-one) [reply communications/shortlinks.csv:12]: I think naming rooms is a good idea, although I'd prefer if the names were more directly applicable, like "comms" or "general", etc.

File: communications/shortlinks.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
+keyword,destination_url
+shortlinks,https://github.com/tomeshnet/toronto-community-network/blob/master/communications/shortlinks.csv
+cloud,https://cloud.tomesh.net
+github,https://github.com/tomeshnet
+gh,https://link.tomesh.net/tomeshnet
```
Comment (makeworld-the-better-one) [inline communications/shortlinks.csv:5]:
```suggestion
gh,https://github.com/tomeshnet
```
Is it too hard to just keep these in sync?

File: communications/shortlinks.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+chat,https://chat.tomesh.net/#/room/#tomesh:tomesh.net
+tasks,https://github.com/tomeshnet/toronto-community-network/projects/1
+new-task,https://github.com/tomeshnet/toronto-community-network/issues/new?template=task_template.md
+teams,https://github.com/orgs/tomeshnet/teams
+passwords,https://pass.tomeshnet.coop
```
Comment (makeworld-the-better-one) [inline communications/shortlinks.csv:10]:
```suggestion
passwords,https://pass.tomesh.net
```

File: communications/shortlinks.csv (hunk 1 above)
Comment (benhylau) [reply communications/shortlinks.csv:12]: Rooms don't always map to WGs, that's why it's hard to associate them. In reality we only need 2-3 rooms, but we alrdy have 4 WGs and a general meeting. I think it's better to use completely unassociated names. We can stick to our tradition of subway stop names :P
tomesh0

Comment (darkdrgn2k) [reply communications/shortlinks.csv:12]: the point of these links to make it easier to find rooms.

Last meetup a bunch of us where in the wrong room and thats with only two rooms

File: communications/shortlinks.csv (hunk 2 above)
Comment (darkdrgn2k) [reply communications/shortlinks.csv:5]: ~*shrug*~ 

Just takes 1 person to forget to update.

**Pull request merged**